spec/unit/puppet/provider/java_ks/keytool_spec.rb

```diff
@@ -245,5 +249,7 @@
   end
 end
 
+# rubocop:disable Lint/EmptyClass
```

Comment:
there seem to be a decent few inline exceptions. Can these not be dealt with in the todo?

Reply:
This is just part of the tests